gstcefsrc.cc

```diff
@@ -21,14 +31,45 @@ GST_DEBUG_CATEGORY_STATIC (cef_console_debug);
 #define DEFAULT_URL "https://www.google.com"
 #define DEFAULT_GPU FALSE
 #define DEFAULT_CHROMIUM_DEBUG_PORT -1
-#define DEFAULT_LOG_SEVERITY LOGSEVERITY_DISABLE
+#define DEFAULT_LOG_SEVERITY LOGSEVERITY_DEFAULT
+#if !defined(__APPLE__) && defined(GST_CEF_USE_SANDBOX)
+#define DEFAULT_SANDBOX TRUE
+#else
 #define DEFAULT_SANDBOX FALSE
+#endif
+
+using CefStatus = enum : guint8 {
+  // CEF was either unloaded successfully or not yet loaded.
+  CEF_STATUS_NOT_LOADED = 0U,
+  // Blocks other elements from initializing CEF is it's already in progress.
+  CEF_STATUS_INITIALIZING = 1U,
+  // CEF's initialization process has completed successfully.
+  CEF_STATUS_INITIALIZED = 1U << 2U,
```

Comment:
I don't think the bit shifts are needed, unless you wanted to use those as flags but that was not exactly what I was suggesting at least :)

Reply:
That was indeed the purpose :) By turning them into flags, I could make masks to be used with the busy loops (just below this line)

Reply:
OK, carry on then :)